code/modules/mob/living/carbon/human/species/slime.dm

```diff
@@ -176,6 +176,16 @@
 	else
 		to_chat(H, "<span class='warning'>You need to hold still in order to regrow a limb!</span>")
 
+
+/obj/item/organ/internal/heart/slime
+	name = "osmatic pressure regulator"
+	desc = "A wet squishy heart."
+
+/obj/item/organ/internal/lungs/slime
```

Comment:
These are defined twice.

Reply:
fixed, I forgot to remove them after I moved them to their own file